deploy/environments/access.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+CODE_ROOT: '/home/dimagi/www/commcare-sync/code_root'
+VIRTUALENV_ROOT: '/home/dimagi/www/.virtualenvs/commcare-sync'
+DJANGO_SETTINGS_MODULE: 'commcare_sync.local'
+DJANGO_PROCESS: 'commcare_sync'
+CELERY_PROCESS: 'commcare_sync_celery_default'
```

Comment:
random thought: now that some of these are more standardized via ansible we could consider making them defaults so they only have to be included if they are "abnormal".

Reply:
Yes, Cory. I agree with you. We need to standardize the home directory for the application code/data like we have for commcare_cloud (user: cchq). thanks